src/components/ProjectCard.js

```diff
@@ -4,81 +4,116 @@ import CardContent from '@material-ui/core/CardContent';
 import Chip from '@material-ui/core/Chip';
 import Grid from '@material-ui/core/Grid';
 import FiberManualRecordRoundedIcon from '@material-ui/icons/FiberManualRecordRounded';
-import githubColorDictionary from './data/gh-colors.json'
+import githubColorDictionary from './data/gh-colors.json';
+import Avatar from '@material-ui/core/Avatar';
 
 const issueStyles = {
   fontSize: '13px',
   fontStyle: 'normal',
   fontWeight: '400',
-}
+};
 
 const renderLanguageChip = (language) => {
-  return <Chip
-    size="small"
-    style={{ backgroundColor: '#F1F1F1' }}
```

Comment:
Would you mind using makeStyles() instead of inline styling. I think in the code style  document we do not want to use  in-line style

Reply:
refactored